df.viewscreen.xml

```diff
@@ -541,14 +541,14 @@
         <int16_t/>
         <int16_t/>
         <int16_t/>
-        <int32_t/>
-        <int32_t/>
-        <int32_t/>
-        <int32_t/>
-        <int32_t/>
-        <int32_t/>
-        <int32_t/>
-        <int8_t/>
+        <int32_t name='x_iterator' comment='0 to map_width-1'/>
+        <int32_t name='y_iterator' comment='0 to map_height-1'/>
+        <int32_t name='rgb_buffer' comment='pointer to allocated buffer'/>
+        <int32_t name='map_width'  comment='in world tiles'/>
+        <int32_t name='map_height' comment='in world tiles'/>
+        <int32_t name='numtiles_map_width' comment='in embark tiles'/>
+        <int32_t name='numtiles_map_height' comment='in embark tiles'/>
+        <int8_t name='enabled' comment='true->exporting enabled'/>
```

Comment:
If this is true when exporting, something like `in_export` or `exporting` might be clearer.